app/controllers/api/database.php

```diff
@@ -1868,7 +1868,7 @@ function createAttribute(string $collectionId, Document $attribute, Response $re
     ->inject('usage')
     ->inject('events')
     ->inject('mode')
-    ->action(function (string $collectionId, string $documentId, string|array $data, ?array $read, ?array $write, Response $response, Database $dbForProject, EventAudit $audits, Stats $usage, Event $events, string $mode) {
+    ->action(function (string $collectionId, string $documentId, string|array|null $data, ?array $read, ?array $write, Response $response, Database $dbForProject, EventAudit $audits, Stats $usage, Event $events, string $mode) {
```

Comment:
```suggestion
    ->action(function (string $collectionId, string $documentId, string|array $data, ?array $read, ?array $write, Response $response, Database $dbForProject, EventAudit $audits, Stats $usage, Event $events, string $mode) {
```
you can remove `null` from the types here , now that you have set the the last argument to true for the param.